### PR TITLE
Accessibility: Added "tab" role to Pivot overflow button 

### PR DESCRIPTION
**Pull request by usmanhf**
## Previous Behavior

When using the Microsoft Accessibility for Web extension's Automated Checks it highlights the Pivot component with the following accessibility issue:
Accessibility bug highlight
bugDescription

## New Behavior

Accessibility issue no longer highlighted:
image

## Related Issue(s)

- Fixes #
  - https://github.com/microsoft/fluentui/issues/27052

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/39a57b2d3d753d2316300ec424e6a2c213887c3d'>39a57b2d3d753d2316300ec424e6a2c213887c3d</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/BaseButton-mount_base.html">806</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/BaseButton-mount.html">814</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Breadcrumb-mount_base.html">2211</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Breadcrumb-mount.html">2190</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Checkbox-mount_base.html">2148</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Checkbox-mount.html">2191</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/CheckboxBase-mount_base.html">1893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/CheckboxBase-mount.html">1904</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ChoiceGroup-mount_base.html">3771</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ChoiceGroup-mount.html">3762</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ComboBox-mount_base.html">893</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ComboBox-mount.html">889</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/CommandBar-mount_base.html">8056</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/CommandBar-mount.html">8054</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ContextualMenu-mount_base.html">16472</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ContextualMenu-mount.html">16736</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DefaultButton-mount_base.html">967</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DefaultButton-mount.html">982</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DetailsRow-mount_base.html">2930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DetailsRow-mount.html">2923</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DetailsRowFast-mount_base.html">2917</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DetailsRowFast-mount.html">2929</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DetailsRowNoStyles-mount_base.html">2690</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DetailsRowNoStyles-mount.html">2686</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Dialog-mount_base.html">3448</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Dialog-mount.html">3470</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DocumentCardTitle-mount_base.html">323</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/DocumentCardTitle-mount.html">338</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Dropdown-mount_base.html">2540</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Dropdown-mount.html">2520</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/FocusTrapZone-mount_base.html">1513</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/FocusTrapZone-mount.html">1522</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/FocusZone-mount_base.html">1454</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/FocusZone-mount.html">1460</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedList-mount_base.html">50455</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedList-mount.html">58712</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedList-virtual-rerender_base.html">24270</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedList-virtual-rerender.html">24407</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">75434</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedList-virtual-rerender-with-unmount.html">75516</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedListV2-mount_base.html">320</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedListV2-mount.html">314</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedListV2-virtual-rerender_base.html">293</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedListV2-virtual-rerender.html">296</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">317</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">320</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/IconButton-mount_base.html">1418</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/IconButton-mount.html">1437</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Label-mount_base.html">450</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Label-mount.html">460</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Layer-mount_base.html">3609</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Layer-mount.html">3632</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Link-mount_base.html">529</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Link-mount.html">526</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/MenuButton-mount_base.html">1220</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/MenuButton-mount.html">1224</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/MessageBar-mount_base.html">27549</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/MessageBar-mount.html">27914</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Nav-mount_base.html">2551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Nav-mount.html">2548</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/OverflowSet-mount_base.html">1029</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/OverflowSet-mount.html">1007</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Panel-mount_base.html">2263</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Panel-mount.html">2252</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Persona-mount_base.html">985</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Persona-mount.html">990</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Pivot-mount_base.html">1189</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Pivot-mount.html">1150</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/PrimaryButton-mount_base.html">1117</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/PrimaryButton-mount.html">1090</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Rating-mount_base.html">5781</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Rating-mount.html">5825</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SearchBox-mount_base.html">1147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SearchBox-mount.html">1165</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Shimmer-mount_base.html">2359</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Shimmer-mount.html">2370</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Slider-mount_base.html">1724</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Slider-mount.html">1726</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SpinButton-mount_base.html">3725</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SpinButton-mount.html">3738</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Spinner-mount_base.html">516</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Spinner-mount.html">519</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SplitButton-mount_base.html">2439</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SplitButton-mount.html">2438</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Stack-mount_base.html">531</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Stack-mount.html">530</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/StackWithIntrinsicChildren-mount_base.html">1127</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/StackWithIntrinsicChildren-mount.html">1134</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/StackWithTextChildren-mount_base.html">3262</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/StackWithTextChildren-mount.html">3221</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SwatchColorPicker-mount_base.html">8149</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/SwatchColorPicker-mount.html">8167</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/TagPicker-mount_base.html">2002</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/TagPicker-mount.html">1988</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Text-mount_base.html">499</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Text-mount.html">498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/TextField-mount_base.html">1207</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/TextField-mount.html">1213</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ThemeProvider-mount_base.html">1114</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ThemeProvider-mount.html">1130</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ThemeProvider-virtual-rerender_base.html">773</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ThemeProvider-virtual-rerender.html">772</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1742</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1725</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Toggle-mount_base.html">789</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/Toggle-mount.html">803</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/buttonNative-mount_base.html">285</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/27691/perf-test/buttonNative-mount.html">277</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

File: change/@fluentui-react-4bfc9233-a921-495f-8652-fd1d9e25f749.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (fabricteam) [inline change/@fluentui-react-4bfc9233-a921-495f-8652-fd1d9e25f749.json:1]: ## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 2 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=27691&commitId=3851a6ed0ba1068d8dc25ab7092d7d8410ef51ba&env=prod)

✅ There was 2 screenshots added, 0 screenshots removed, 1045 screenshots unchanged, 0 screenshots with different dimensions and 0 screenshots with visible difference.


  <details><summary><b>unknown</b> 2 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Pivot - Overflow.Tabs - RTL.Narrow - Last tab selected.chromium.png|0| Added|
|Pivot - Overflow.Tabs - RTL.Narrow - Overflow menu.chromium.png|0| Added|

</details>


<div id="vrtCommentfluentuiv8"/>

**Review by sopranopillow — APPROVED** [review]

File: change/@fluentui-react-4bfc9233-a921-495f-8652-fd1d9e25f749.json (hunk 1 above)
Comment (usmanhf) [reply change/@fluentui-react-4bfc9233-a921-495f-8652-fd1d9e25f749.json:1]: Resolved because visual regression passed after changes ran

**Pull request closed without merging**